src/common/meta/src/ddl_manager.rs

```diff
@@ -14,18 +14,382 @@
+    TruncateTableTask,
+};
+use crate::rpc::router::RegionRoute;
+
+pub type DdlManagerRef = Arc<DdlManager>;
+
+pub struct DdlManager {
+    procedure_manager: ProcedureManagerRef,
+    datanode_manager: DatanodeManagerRef,
+    cache_invalidator: CacheInvalidatorRef,
+    table_metadata_manager: TableMetadataManagerRef,
+    table_creator: TableCreatorRef,
+}
+
+impl DdlManager {
+    pub fn new(
+        procedure_manager: ProcedureManagerRef,
+        datanode_clients: DatanodeManagerRef,
+        cache_invalidator: CacheInvalidatorRef,
+        table_metadata_manager: TableMetadataManagerRef,
+        table_creator: TableCreatorRef,
+    ) -> Self {
+        Self {
+            procedure_manager,
+            datanode_manager: datanode_clients,
+            cache_invalidator,
+            table_metadata_manager,
+            table_creator,
+        }
+    }
+
+    pub fn table_metadata_manager(&self) -> &TableMetadataManagerRef {
+        &self.table_metadata_manager
+    }
+
+    pub fn create_context(&self) -> DdlContext {
+        DdlContext {
+            datanode_manager: self.datanode_manager.clone(),
+            cache_invalidator: self.cache_invalidator.clone(),
+            table_metadata_manager: self.table_metadata_manager.clone(),
+        }
+    }
+
+    pub fn try_start(&self) -> Result<()> {
+        let context = self.create_context();
+
+        self.procedure_manager
+            .register_loader(
+                CreateTableProcedure::TYPE_NAME,
+                Box::new(move |json| {
+                    let context = context.clone();
+                    CreateTableProcedure::from_json(json, context).map(|p| Box::new(p) as _)
+                }),
+            )
+            .context(RegisterProcedureLoaderSnafu {
+                type_name: CreateTableProcedure::TYPE_NAME,
+            })?;
+
+        let context = self.create_context();
+
+        self.procedure_manager
+            .register_loader(
+                DropTableProcedure::TYPE_NAME,
+                Box::new(move |json| {
+                    let context = context.clone();
+                    DropTableProcedure::from_json(json, context).map(|p| Box::new(p) as _)
+                }),
+            )
+            .context(RegisterProcedureLoaderSnafu {
+                type_name: DropTableProcedure::TYPE_NAME,
+            })?;
+
+        let context = self.create_context();
+
+        self.procedure_manager
+            .register_loader(
+                AlterTableProcedure::TYPE_NAME,
+                Box::new(move |json| {
+                    let context = context.clone();
+                    AlterTableProcedure::from_json(json, context).map(|p| Box::new(p) as _)
+                }),
+            )
+            .context(RegisterProcedureLoaderSnafu {
+                type_name: AlterTableProcedure::TYPE_NAME,
+            })
+    }
+
+    pub async fn submit_alter_table_task(
```

Comment:
I think we need to print some logs to record all DDL changes.
We can print info log at the beginning of handleXXX or submitXXX method.